chore: make mixin range interval configurable

jmichalek132 opened the pull request:
**What this PR does / why we need it**:

Address last recording rule which has a hardcoded range interval which is too small for scrape interval of 1m by making it configurable using existing variables.

**Which issue(s) this PR fixes**:
Fixes #13630

**Special notes for your reviewer**:

**Checklist**
- [x] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [ ] Documentation added
- [ ] Tests updated
- [ ] Title matches the required conventional commits format, see [here](https://www.conventionalcommits.org/en/v1.0.0/)
  - **Note** that Promtail is considered to be feature complete, and future development for logs collection will be in [Grafana Alloy](https://github.com/grafana/alloy). As such, `feat` PRs are unlikely to be accepted unless a case can be made for the feature actually being a bug fix to existing behavior.
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/setup/upgrade/_index.md`
- [ ] For Helm chart changes bump the Helm chart version in `production/helm/loki/Chart.yaml` and update `production/helm/loki/CHANGELOG.md` and `production/helm/loki/README.md`. [Example PR](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213)
- [ ] If the change is deprecating or removing a configuration option, update the `deprecated-config.yaml` and `deleted-config.yaml` files respectively in the `tools/deprecated-config-checker` directory. [Example PR](https://github.com/grafana/loki/pull/10840/commits/0d4416a4b03739583349934b96f272fb4f685d15)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/grafana/loki?pullRequest=13925) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/grafana/loki?pullRequest=13925) before we can accept your contribution.<br/><hr/>**Juraj Michalek** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/grafana/loki?pullRequest=13925) it.</sub>

cstyan commented:
hey @jmichalek132 thanks for this PR, but unfortunately right now this won't work as expected because we're still on an older version of the `mixin-utils` library that the histogram function is pulled from. The version we're on doesn't take in any parameter for the interval.

Can you run `jb update` and then generate the dashboards again?

jmichalek132 commented:
> hey @jmichalek132 thanks for this PR, but unfortunately right now this won't work as expected because we're still on an older version of the `mixin-utils` library that the histogram function is pulled from. The version we're on doesn't take in any parameter for the interval.
> 
> Can you run `jb update` and then generate the dashboards again?

Yeah, sorry my bad I lost that change as I switched devices to work on the PR. Fixed now.

cstyan commented:
@jmichalek132 thanks!
